Analysis/LeptonID/include/PIDutil.h

```diff
@@ -0,0 +1,33 @@
+#include <lcio.h>
+#include <EVENT/Track.h>
+#include <EVENT/Cluster.h>
+#include <EVENT/MCParticle.h>
+#include <EVENT/Vertex.h>
+#include <UTIL/LCRelationNavigator.h>
+#include <marlinutil/WeightedPoints3D.h>
+#include <TVector3.h>
+
+using namespace lcio;
+
+float getCorrectdEdx(float dEdx, Track *track);
+
+LCObject *getRelated(LCObject *p, LCRelationNavigator const &nav, int &_cw, int &_tw);
```

Comment:
```suggestion
std::tuple<LCObject*, float, float> getRelated(LCObject *p, LCRelationNavigator const &nav);
```
Would be a possibility to get rid of the output reference parameters. Since we are using c++17 in any case it is also easy to destructure this with structured bindings on the calling site:
```cpp
/*const*/ auto [obj, track_w, cluster_w] = getRelated(/* */);
```


In general, it looks like the weights should be `float` and not `int` unless you specifically want them to be in permille. (You seem to indicate that in the implementation).